bindings/src/main/java/io/pravega/storage/extendeds3/ExtendedS3Storage.java

```diff
@@ -13,12 +13,7 @@
 import com.emc.object.s3.S3Client;
 import com.emc.object.s3.S3Exception;
 import com.emc.object.s3.S3ObjectMetadata;
-import com.emc.object.s3.bean.AccessControlList;
-import com.emc.object.s3.bean.CanonicalUser;
-import com.emc.object.s3.bean.CopyPartResult;
-import com.emc.object.s3.bean.Grant;
-import com.emc.object.s3.bean.MultipartPartETag;
-import com.emc.object.s3.bean.Permission;
+import com.emc.object.s3.bean.*;
```

Comment:
The checkStyle rule will fail on this commit.
Style rule prohibits using * imports. 
I suggest you run `./gradlew compileJava compileTest checkStyleMain checkStyleTest spotBugsMain spotbugsTest` to find all issues and fix.  
(In IdeaJ when you hover over * import the "lighbulb" hint shows option to replace with single imports.)
Please fix the issues.